logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/mlsmigration/MLSMigrationManager.kt

```diff
@@ -73,7 +73,7 @@ internal class MLSMigrationManagerImpl(
             incrementalSyncRepository.incrementalSyncState.collect { syncState ->
                 ensureActive()
                 if (syncState is IncrementalSyncStatus.Live &&
-                    featureSupport.isMLSSupported &&
+                    isMLSEnabledUseCase() &&
```

Comment:
```suggestion
                    isMLSEnabled() &&
```

Just a tiny change.

Reply:
Thanks for the note, I'm afraid if I change it now; then in other cherrypicks gonna create other issues!